### PR TITLE
Add harbor disableRedirect schema description

### PR DESCRIPTION
**Pull request by anders-elastisys**
> [!warning]
> **This is a public repository, ensure not to disclose:**
>
> - [x] personal data beyond what is necessary for interacting with this pull request, nor
> - [x] business confidential information, such as customer names.

### What kind of PR is this?

**Required**: Mark one of the following that is applicable:

- [ ] kind/feature       
- [x] kind/improvement   
- [ ] kind/deprecation   
- [ ] kind/documentation 
- [ ] kind/clean-up      
- [ ] kind/bug           
- [ ] kind/other         

*Optional*: Mark one or more of the following that are applicable:

> [!important]
> Breaking changes should be marked `kind/admin-change` or `kind/dev-change` depending on type
> Critical security fixes should be marked with `kind/security`

- [ ] kind/admin-change 
- [ ] kind/dev-change   
- [ ] kind/security     
- [ ] [kind/adr]()      

### What does this PR do / why do we need this PR?

Updates the schema for Harbor `disableRedirect` config.

- Fixes #

#### Checklist

- [ ] Proper commit message prefix on all commits
  
- Change checks:
  - [ ] The change is transparent
  - [ ] The change is disruptive
  - [ ] The change requires no migration steps
  - [ ] The change requires migration steps
  - [ ] The change upgrades CRDs
  - [ ] The change updates the config *and* the schema
- Documentation checks:
  - [ ] The [public documentation](https://github.com/elastisys/compliantkubernetes) required no updates
  - [ ] The [public documentation](https://github.com/elastisys/compliantkubernetes) required an update - [link to change](set-me\)
- Metrics checks:
  - [ ] The metrics are still exposed and present in Grafana after the change
  - [ ] The metrics names didn't change (Grafana dashboards and Prometheus alerts are not affected)
  - [ ] The metrics names did change (Grafana dashboards and Prometheus alerts were fixed)
- Logs checks:
  - [ ] The logs do not show any errors after the change
- Pod Security Policy checks:
  - [ ] Any changed pod is covered by Pod Security Admission
  - [ ] Any changed pod is covered by Gatekeeper Pod Security Policies
  - [ ] The change does not cause any pods to be blocked by Pod Security Admission or Policies
- Network Policy checks:
  - [ ] Any changed pod is covered by Network Policies
  - [ ] The change does not cause any dropped packets in the `NetworkPolicy Dashboard`
- Audit checks:
  - [ ] The change does not cause any unnecessary Kubernetes audit events
  - [ ] The change requires changes to Kubernetes audit policy
- Falco checks:
  - [ ] The change does not cause any alerts to be generated by Falco
- Bug checks:
  - [ ] The bug fix is covered by regression tests

### CODE REVIEW

File: config/schemas/config.yaml [hunk 1]
```diff
@@ -1923,7 +1923,12 @@ properties:
         additionalProperties: false
         properties:
           disableRedirect:
+            title: Harbor registry disable ObjectStorage redirect
             type: boolean
+            description: |-
+              Controls whether or not Harbor registry redirects users to the ObjectStorage endpoint.
+              Set this to true if the ObjectStorage is not reachable by users when pushing images to Harbor.
```
Comment (cristiklein) [inline config/schemas/config.yaml:1930]: Can you copy-paste the error message that the user would see either in `docker pull`, `docker push` or `kubectl describe pod`?

**Review by cristiklein — CHANGES_REQUESTED** [review]
Thanks for improving the configuration documentation. Can you also add the error message to make this even more useful?

File: config/schemas/config.yaml (hunk 1 above)
Comment (anders-elastisys) [reply config/schemas/config.yaml]: PTAL

**Review by cristiklein — APPROVED** [review]
Perfect, thank you!

File: config/schemas/config.yaml [hunk 2]
```diff
@@ -1923,7 +1923,17 @@ properties:
         additionalProperties: false
         properties:
           disableRedirect:
+            title: Harbor registry disable ObjectStorage redirect
```
Comment (aarnq) [inline config/schemas/config.yaml:1926]: Nit
```suggestion
            title: Harbor registry disable object storage redirect
```
Same for the description.

**Pull request merged**